crates/binstalk-downloader/src/remote.rs

```diff
@@ -279,11 +279,9 @@ impl Client {
         }
     }
 
-    /// Check if remote exists using `Method::HEAD` or `Method::GET` as fallback.
+    /// Check if remote exists using `Method::GET` as fallback.
```

Comment:
...sorry but there's no "as fallback" anymore either ^^

Reply:
Thanks, I've also fixed this.